core/vm/operations_acl.go

```diff
@@ -21,8 +21,8 @@ import (
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/common/math"
+	"github.com/ethereum/go-ethereum/core/state"
 	"github.com/ethereum/go-ethereum/params"
-	"github.com/ethereum/go-ethereum/trie/utils"
 )
 
 func makeGasSStoreFunc(clearingRefund uint64) gasFunc {
```

Comment:
Wouldn't any EIP that calls `makeGasSStoreFunc` be overriden by the new `enable4762` that overwrites the dynamigGas of SSTORE with `gasSStore4762(...)` (L89) below?

As in, do we need to do these changes in this method?
Maybe I'm a bit lost here.

Reply:
yes it would, but because I set all constant gas costs to 0 I have to change the return values here to make it work in 2929.